Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <TargetFrameworks>net6.0;netstandard2.0;net471</TargetFrameworks>
+	  <LangVersion>latest</LangVersion>
+	  <ImplicitUsings>enable</ImplicitUsings>
+	  <IncludeSource>true</IncludeSource>
+	  <AssemblyName>Frends.SFTP.UploadFiles</AssemblyName>
+	  <RootNamespace>Frends.SFTP.UploadFiles</RootNamespace>
+
+	  <Version>0.0.15</Version>
```

Comment:
So... why?.. :)

Reply:
This was to get task with version under 1.0.0 to my tenant to test this. Fixed now.